go.mod

```diff
@@ -512,7 +512,7 @@ replace (
 	k8s.io/klog/v2 => k8s.io/klog/v2 v2.8.0
 	k8s.io/kube-aggregator => ./staging/src/k8s.io/kube-aggregator
 	k8s.io/kube-controller-manager => ./staging/src/k8s.io/kube-controller-manager
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
+	k8s.io/kube-openapi => github.com/gautierdelorme/kube-openapi v0.0.0-20210418101956-5b3ab66456a2
```

Comment:
I'm sure that's a mistake ;-)

Reply:
@apelisse this is a WIP PR 😊 Obviously I'll use kubernetes/kube-openapi once the other PR is merged. I linked to the two blockers in the PR description.